package.json

```diff
@@ -16,19 +16,16 @@
     "README.md",
     "dist"
   ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/wowserhq/wowser-math.git"
-  },
+  "repository": "wowserhq/wowser-math",
```

Comment:
I'm cool with this, although I'd propose we use this syntax (to be less ambiguous):

```
"repository": "github:wowserhq/wowser-math"
```

See: https://docs.npmjs.com/files/package.json#repository

Reply:
Nice, will fixed 👍